### PR TITLE
6724 - Added validateForm and clearDate to DatePicker

### PR DESCRIPTION
**Pull request by andrepestana-aot**
*Issue #:* /bcgov/entity#6724

*Description of changes:*

- Added public validateForm method
- Changed clearDate visibility to public
- Added tests

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the business-filings-ui license (Apache 2.0).

### CODE REVIEW

File: src/components/DatePicker/package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@bcrs-shared-components/date-picker",
-  "version": "1.1.0",
+  "version": "1.1.1",
```
Comment (cameron-freshworks) [inline src/components/DatePicker/package.json:3]: You will want to revert this, as Lerna handles the versioning :)

**Review by cameron-freshworks — COMMENTED** [review]
Looks good! Just want to revert the versioning as that will be done after we merge into Master. 

That being said, the process is a little different on the shared repo. We all commit from a branch within the main repo, not a FORK in this case. 
I'm not sure, but we might have tag/version issues if we are committing from a FORK.
@severinbeauvais or @vysakh-menon-aot might know best on that. 

Code looks good though! 👍

**Review by severinbeauvais — COMMENTED** [review]
LGTM (other than please revert version increase, per Cam's comment).

**Comment by andrepestana-aot** [comment]
> LGTM (other than please revert version increase, per Cam's comment).

ok. Should I create a branch from it instead of from a fork?

**Comment by andrepestana-aot** [comment]
Closing this one and creating a new one from a branch.

**Pull request closed without merging**